src/libraries/Microsoft.Extensions.DependencyInjection/src/ServiceLookup/Expressions/ExpressionResolverBuilder.cs

```diff
@@ -156,8 +156,8 @@ protected override Expression VisitDisposeCache(ServiceCallSite callSite, object
                 VisitCallSiteMain(callSite, context));
         }
 
-        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2060:UnrecognizedReflectionPattern",
-            Justification = "Calling Array.Empty<T>() is safe since the T doesn't have linker annotations.")]
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2060:MakeGenericMethod",
+            Justification = "Calling Array.Empty<T>() is safe since the T doesn't have ILLink annotations.")]
```

Comment:
```suggestion
            Justification = "Calling Array.Empty<T>() is safe since the T doesn't have trimming annotations.")]
```

Let's avoid referring to the tool name if possible. The choice of the tool name is bad because [linker](https://en.wikipedia.org/wiki/Linker_(computing)) in computing means a different thing than what ILLink does. It would be good not to leak it into class libraries.